README.md

```diff
@@ -274,8 +280,8 @@ The `exclude` option has the following defaults settings:
 These settings exclude `test` and `__tests__` directories as well as `test.js`, `*.test.js`, and `test-*.js` files.
 Specifying your own exclude property completely replaces these defaults.
 
-**Note:** The exclude list always implicitly contains `**/node_modules/**`, even if not explicitly specified in an overriding `exclude` array.
-To reverse this you must add the negated exclude rule `!**/node_modules/`.
+**Note:** We always add `**/node_modules/**` to the exclude list, even if not specified in the config.
+You can override this by setting the `!**/node_modules/` negated exclude rule.
```

Comment:
I've reopened #912.  By the time we release 14.0.0 the correct way to handle this will be to use `--exclude-node-modules=false`.  I'd like for this documentation change to reflect #912 being merged.  So for example you might have nyc.config.js as:
```js
module.exports = {
  all: true
  include: [
    'lib/**',
    'node_modules/@my-org/**'
  ],
  exclude: [
    'node_modules/@my-org/something/unwanted.js',
    '**/test/**'
  ],
  excludeNodeModules: false
}
```
This configuration would instrument everything in `lib/` and `node_modules/@my-org/`, ignore all other node_modules (due to not being included rather than being excluded.  Negated excludes are processed last so `!**/node_modules/**` makes it impossible to exclude any specific files matching that negated exclude.  So `exclude: ['!**/node_modules/@my-org/**', 'node_modules/@my-org/something/unwanted.js']` would not exclude unwanted.js.